CMakeLists.txt

```diff
@@ -523,6 +523,62 @@ if(CIRCT_BINDINGS_TCL_ENABLED)
   message(STATUS "Found TCL executable: ${TCL_TCLSH}")
 endif()
 
+#-------------------------------------------------------------------------------
+# slang Verilog Frontend
+#-------------------------------------------------------------------------------
+
+option(CIRCT_SLANG_FRONTEND_ENABLED "Enables slang Verilog frontend." ON)
```

Comment:
Maybe start with this default to off?

Reply:
Yeah I can definitely do that. And enable it explicitly in CI such that it's covered.

Reply:
Yep that sounds reasonable to me.

Reply:
Done